### PR TITLE
[MetaSchedule] Modify Profiler Timers

### PR DESCRIPTION
**Pull request by zxybazh**
Minor modification to scoped timers to cover 99% of all the time cost during MS tuning. Allow `ApplyHistoryBest` and `TaskExtraction` time to be counted during tune_relay.

### CODE REVIEW

**Review by junrushao1994 — APPROVED** [review]
LGTM!

File: src/meta_schedule/profiler.cc [hunk 1]
```diff
@@ -79,7 +80,8 @@ PackedFunc ProfilerTimedScope(String name) {
                                     tik = std::chrono::high_resolution_clock::now(),  //
                                     name = std::move(name)]() {
       auto tok = std::chrono::high_resolution_clock::now();
-      double duration = std::chrono::duration_cast<std::chrono::seconds>(tok - tik).count();
+      double duration =
+          std::chrono::duration_cast<std::chrono::nanoseconds>(tok - tik).count() / 1e9;
```
Comment (junrushao1994) [inline src/meta_schedule/profiler.cc:84]: quick question: what's the diff between the two?

Comment (zxybazh) [reply src/meta_schedule/profiler.cc:84]: Turns out the previous one would create some inaccuracy in time counting, thus impacting the final result.

Comment (junrushao1994) [reply src/meta_schedule/profiler.cc:84]: Thanks for sharing!

**Pull request merged**